Tests/Common/TestUtils.cs

```diff
@@ -302,18 +304,39 @@ public static void StartBasicsStation(TestConfiguration config, Dictionary<strin
             }
         }
 
-        public static DataRateIndex GetDataRateIndex(this Region region, DataRate datr) =>
-             region.DRtoConfiguration.FirstOrDefault(x => x.Value.DataRate == datr) is (var index, (not null, _)) ? index : throw new KeyNotFoundException("");
-
         public static RadioMetadata GenerateTestRadioMetadata(
                 DataRateIndex dataRate = DataRateIndex.DR2,
                 Hertz? frequency = null,
                 uint antennaPreference = 1,
                 ulong xtime = 100000,
-                uint gpstime = 0,
+                uint gpstime = 100000,
```

Comment:
Why this change?

Reply:
<img width="710" alt="image" src="https://user-images.githubusercontent.com/8555833/146774528-4b01e219-e0c4-4bea-9c4b-a962706d96df.png">
As it's time after Epoch, I imagined 0 was very unlikely but so is any other values